app/views/ordertimes/_form.html.haml

```diff
@@ -42,8 +42,8 @@
     .col-md-1
       = f.text_field(:to_end_time, value: format_time(@worktime.to_end_time))
 
-  = f.labeled_boolean_field(:billable)
-  = f.labeled_boolean_field(:meal_compensation)
+  = f.labeled_boolean_field(:billable, detail: 'Leistung ist verrechenbar.')
+  = f.labeled_boolean_field(:meal_compensation, detail: "Ich war vor Ort beim Kunden und habe Anrecht auf die #{link_to('Verpflegungsentschädigung', 'https://wiki.puzzle.ch/Puzzle/Getr%c3%a4nkeentsch%c3%a4digung')}.".html_safe)
```

Comment:
Gibt es einen gescheiten Weg wie man sowas konfigurierbar machen könnten? Weil Site spezifische Links in einem Open Source Projekt sollten wir möglichst vermeiden aus meiner Sicht.

Reply:
Ich hatte noch vor, dies eventuell in ein Language File zu packen. Aber deine Lösung gefällt mir besser.